edgePy/DGEList.py

```diff
@@ -28,17 +28,23 @@ class DGEList(object):
         filename: a shortcut to import NPZ (zipped numpy format) files.
 
     Examples:
-        >>> from smart_open import smart_open
+
         >>> from edgePy.data_import import get_dataset_path
         >>> dataset = 'GSE49712_HTSeq.txt.gz'
-        >>> DGEList.read_handle(smart_open(get_dataset_path(dataset), 'r'))
-        DGEList(num_samples=10, num_genes=21,716)
+        >>> group_file = 'groups.json'
+        >>> DGEList.create_DGEList_data_file(get_dataset_path(dataset), get_dataset_path(group_file))
+        DGEList(num_samples=10, num_genes=21,711)
 
     """
 
     # Pattern to delete from field names anytime they are assigned.
     _field_strip_re = re.compile(r'[\s"]+')
 
+    # Metatags used in older HTSeq datasets without underscore prefixes.
+    _old_metatags = np.array(
+        ['no_feature', 'ambiguous', 'too_low_aQual', 'not_aligned', 'alignment_not_unique']
+    )
+
```

Comment:
Inconsistent np.array structure compared to in tests/test_DGEList.py

Reply:
Can't fix - this is how black does the format.